english/dream/tender-defender/tender-defender.md

```diff
@@ -0,0 +1,28 @@
+A woman benefits from a kind, tender partner who cares for her and a strong, decisive protector who can defend her.
+If you can do both - if you are a 'tender defender' - you become the whole package. 
+
+Tender defender dos and don'ts:
+
+DO BE: empathetic
+DON'T BE: narcissistic
+
+DO BE: assertive
+DON'T BE: wimpy
+
+DO BE: protective
+DON'T BE: psychopathic
+
+DO BE: generous leader
+DON'T BE: machiavellian
+
+DO BE: altruistic
+DON'T BE: selfish
+
+DO BE: confident and decisive
+DON'T BE: submissive or passive
+
+DO BE: ready for anything
+DON'T BE: completely oblivious or totally anxious
+
+If you can do all these things, you will be on your way to becoming 
+[what women want](https://www.amazon.com/Mate-Become-Man-Women-Want-ebook/dp/B00RTY0FKK)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.